<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="PicLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>491</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BeforeLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AfterLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>158</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="BeforeBtn">
       <property name="text">
        <string>上一张</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AfterBtn">
       <property name="text">
        <string>下一张</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>450</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>中文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英文</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="ErZhiSlider">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="GaMaSlider">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="GaMalabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>伽马变化</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GaMalineEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>310</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ErZhilabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>二值化</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ErZhilineEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LanguageLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>语言</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>文件(&amp;O)</string>
    </property>
    <addaction name="OpenAct"/>
   </widget>
   <widget class="QMenu" name="LanguageMenu">
    <property name="title">
     <string>语言(&amp;L)</string>
    </property>
    <addaction name="ChineseAct"/>
    <addaction name="EnglishAct"/>
   </widget>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="LanguageMenu"/>
   <addaction name="HelpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_3">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>77</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="OpenBtn">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="YuanTuBtn">
        <property name="text">
         <string>显示原图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HuiDuBtn">
        <property name="text">
         <string>灰度化</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LvBoBtn">
        <property name="text">
         <string>3*3均值滤波</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BianYuanBtn">
        <property name="text">
         <string>边缘检测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveAsBtn">
        <property name="text">
         <string>另存为</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="OpenAct">
   <property name="text">
    <string>打开(&amp;F)</string>
   </property>
  </action>
  <action name="ChineseAct">
   <property name="text">
    <string>中文(&amp;C)</string>
   </property>
  </action>
  <action name="EnglishAct">
   <property name="text">
    <string>英文(&amp;E)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
